contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -44,7 +44,7 @@ contract IpRoyaltyVault is IIpRoyaltyVault, ERC20Upgradeable, ReentrancyGuardUpg
         mapping(uint256 snapshotId => mapping(address claimer => mapping(address token => bool))) isClaimedAtSnapshot;
         EnumerableSet.AddressSet tokens;
         mapping(address token => uint256 accBalance) poolInfo;
-        mapping(address token => mapping(address claimer => uint256 revenueDebt)) claimerInfo;
+        mapping(address token => mapping(address claimer => int256 revenueDebt)) claimerInfo;
```

Comment:
why the change from uint256 to int256?

Reply:
it because the `revenueDebt` could be negative value which indicate there is pending revenue of claimer.